<?xml version="1.0" encoding="utf-8"?>
<report>
	<activity
		result="Failed"
		duration="0ms"
		durationms="0"
		type="root"
		rid="c8111a6c93c5687"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		user="automation"
		host="DESKTOP-IP0304R"
		rxversion="5.4.0.24580"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		language="en-US"
		screenresolution="1280x1024"
		timestamp="3/7/2016 8:15:00 PM"
		timestampiso="2016-03-08T01:15:00"
		timeoutfactor="1"
		module="DisplayQAJobs">
		<detail>
			DisplayQAJobs
		</detail>
		<varbindings/>
		<item
			timeRelativeToTestSuiteStartTime="00:55.078"
			timeRelativeToTestModuleStartTime="00:55.078"
			timeWallClock="20:15:55"
			time="00:55.078"
			level="Error"
			category="Module">
			<message>
				Test run was aborted by the user.
			</message>
			<metainfo
				loglvl="Error"/>
		</item>
	</activity>
</report>
